addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -1019,6 +1006,127 @@ void shouldGenerateValidXmlReport() throws Exception {
         assertThat(alertItemNodes.item(i).getTextContent(), is(equalTo("0")));
         i++;
         assertThat(alertItemNodes.item(i).getNodeName(), is(equalTo("#text"))); // Filler
+        if (!isXmlPlus) {
+            return;
+        }
+        i++;
+        assertThat(alertItemNodes.item(i).getNodeName(), is(equalTo("tags")));
+        i++;
+        assertThat(alertItemNodes.item(i).getNodeName(), is(equalTo("#text"))); // Filler
+    }
+
+    private static void checkXmlAlertInstance(
```

Comment:
Same here to split.

Reply:
For Json and JsonPlus I have splitted. But it will be very messy if I do the same with xml and xmlPlus because `isXmlPlus` is  being used at many random places.